ansible/vars/topo_t1-str.yml

```diff
@@ -0,0 +1,342 @@
+topology:
```

Comment:
can we rename this topology to bt1?

Reply:
Some test cases might select between keyword `t0` and `t1`, renaming to `bt1` will fail those test cases.

Reply:
let's rename to backend-t1

Reply:
can we make it `t1-backend`, so that this won't break the current naming convention?